src/Scrabble/Board/Board.hs

```diff
@@ -39,7 +39,7 @@ import Scrabble.Board.Square
 
 data Board = Board {
   contents :: (Array Point Square)
-} deriving (Eq, Ord, Generic)
+} deriving (Eq, Ord, Generic, Show)
```

Comment:
Okay thanks. At some point I'll go back and document things much better,
but just don't have time yet.

On Fri, Apr 1, 2016 at 2:56 PM, Nick Lawler <notifications@github.com>
wrote:

> In src/Scrabble/Board/Board.hs
> <https://github.com/joshcough/Scrabble/pull/7#discussion_r58251317>:
>
> > @@ -39,7 +39,7 @@ import Scrabble.Board.Square
> >
> >  data Board = Board {
> >    contents :: (Array Point Square)
> > -} deriving (Eq, Ord, Generic)
> > +} deriving (Eq, Ord, Generic, Show)
>
> Ah, hadn't come across showBoard. I'll remove this
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/joshcough/Scrabble/pull/7/files/362506855d92b3542f4c5d90e6044e75aec31332#r58251317>
>